client/src/app/queries/applications.ts

```diff
@@ -34,6 +36,13 @@ export const useFetchApplications = () => {
       queryClient.invalidateQueries([reviewsQueryKey]);
       queryClient.invalidateQueries([assessmentsQueryKey]);
     },
+    select: (apps) =>
+      apps.map(
+        (app: Application): Application => ({
+          ...app,
+          assessments: [mockQuestionnaire],
```

Comment:
Should we add some TODO comments around the usages of `mockQuestionnaire`? I guess the word "mock" in the name is probably enough